### PR TITLE
refactor-cli

### PR DESCRIPTION
**Pull request by invocamanman**

### CODE REVIEW

File: rollup-cli/src/actions/offchain/send.js [hunk 1]
```diff
@@ -34,12 +35,13 @@ function _addNonce(nonceObject, currentBatch, nonce, fromId) {
...
  * @param idTo receiver
- * @param amount initial balance on balance tree
- * @param walletJson from this one can obtain the ethAddress and babyPubKey
- * @param password for decrypt the Wallet
+ * @param amount amount to transfer
+ * @param walletRollup ethAddress and babyPubKey together
  * @param tokenId token type identifier, the sender and the reciever must use the same token
  * @param userFee amount of fee that the user is willing to pay
- * @param idFrom Self balance tree identifier
+ * @param fromId Self balance tree identifier
```
Comment (krlosMata) [inline rollup-cli/src/actions/offchain/send.js]: `fromId` -->   `idFrom` 
just to have correlation among other files

File: rollup-cli/src/actions/offchain/send.js [hunk 2]
```diff
@@ -34,12 +35,13 @@ function _addNonce(nonceObject, currentBatch, nonce, fromId) {
...
+ * @param nonce Hardcoded from user
+ * @param nonceObejct Stored object wich keep tracking of current nonces send by the client
```
Comment (krlosMata) [inline rollup-cli/src/actions/offchain/send.js]: `send` --> `sent` ( if I understood the description well xD )

File: rollup-cli/tools/gunners/info.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Gunners
+
+Automatic creation of deposits and off-chain transactions in order to stress the operator and get performance mesures.
```
Comment (krlosMata) [inline rollup-cli/tools/gunners/info.md]: `mesures` --> `measures`

File: rollup-cli/src/actions/onchain/deposit-and-transfer.js [hunk 4]
```diff
@@ -8,12 +8,14 @@ const { getGasPrice } = require('./utils');
...
  * @param loadAmount initial Amount on balance tree
+ * @param amount amount to transfer
  * @param tokenId token type identifier
- * @param walletJson from this one can obtain the ethAddress and babyPubKey
- * @param passphrase for decrypt the Wallet
+ * @param walletRollup ethAddress and babyPubKey together
  * @param ethAddress allowed address to control new balance tree leaf
  * @param abi abi of rollup contract
- * @param UrlOperator URl from operator
+ * @param toId receiver
```
[the remark this answers is not in the document]
Comment (invocamanman) [reply rollup-cli/src/actions/onchain/deposit-and-transfer.js]: changed!

File: rollup-cli/src/actions/onchain/transfer.js [hunk 5]
```diff
@@ -9,13 +9,15 @@ const { getGasPrice } = require('./utils');
  * @param addressSC rollup address
  * @param amount initial balance on balance tree
  * @param tokenId token type identifier
- * @param walletJson from this one can obtain the ethAddress and babyPubKey
- * @param passphrase for decrypt the Wallet
+ * @param walletRollup ethAddress and babyPubKey together
  * @param abi abi of rollup contract
- * @param UrlOperator URl from operator
+ * @param idFrom sender
+ * @param Idto receiver
```
[the remark this answers is not in the document]
Comment (invocamanman) [reply rollup-cli/src/actions/onchain/transfer.js]: changed!

File: rollup-cli/src/actions/onchain/deposit-and-transfer.js [hunk 6]
```diff
@@ -8,12 +8,14 @@ const { getGasPrice } = require('./utils');
  * @param nodeEth URL of the ethereum node
```
[the remark this answers is not in the document]
Comment (invocamanman) [reply rollup-cli/src/actions/onchain/deposit-and-transfer.js]: updated

**Pull request merged**